packages/serverless-logic-web-tools/src/settings/runtimeTools/RuntimeToolsSettings.tsx

```diff
@@ -60,10 +62,12 @@ export function RuntimeToolsSettings(props: SettingsPageProps) {
   const settingsDispatch = useSettingsDispatch();
   const [config, setConfig] = useState(settings.runtimeTools.config);
   const [isModalOpen, setIsModalOpen] = useState(false);
-  const [isDataIndexUrlValidated, setDataIndexUrlValidated] = useState(DataIndexValidation.INVALID);
+  const [isDataIndexUrlValidated, setDataIndexUrlValidated] = useState(FormValiationOptions.INITIAL);
+  const [isDataIndexUrlVerified, setisDataIndexUrlVerified] = useState(FormValiationOptions.INITIAL);
```

Comment:
Similarily to https://github.com/kumaradityaraj/kie-tools/blob/2709ce4349c467ffba41e2e8f9cbe78c10502333/packages/serverless-logic-web-tools/src/settings/openshift/OpenShiftSettingsSimpleConfig.tsx#L58
Can we use one state, for the validation result?
```suggestion
  const [isConfigValidated, setConfigValidated] = useState(FormValiationOptions.INITIAL);
```